weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionsPage.tsx

```diff
@@ -13,6 +24,8 @@ import {LoadingDots} from '../../../../LoadingDots';
 import {Timestamp} from '../../../../Timestamp';
 import {useWeaveflowRouteContext} from '../context';
 import {StyledDataGrid} from '../StyledDataGrid';
+import {buildDynamicColumns} from './CallsPage/callsTableColumns';
```

Comment:
It would be good to move these newly generic functions to a location that didn't have "CallsPage"  / "callsTableColumns" in the name.

Reply:
great point. Moving to a shared location